react/FilePicker/FilePickerBody.jsx

```diff
@@ -99,14 +100,11 @@ const FilePickerBody = ({
 }
 
 FilePickerBody.propTypes = {
-  onSelectFileId: PropTypes.func.isRequired,
-  filesIdsSelected: PropTypes.arrayOf(PropTypes.string).isRequired,
+  onSelectItemId: PropTypes.func.isRequired,
+  itemsIdsSelected: PropTypes.arrayOf(PropTypes.string).isRequired,
   folderId: PropTypes.string.isRequired,
   navigateTo: PropTypes.func.isRequired,
-  fileTypesAccepted: PropTypes.exact({
-    file: PropTypes.bool,
-    folder: PropTypes.bool
-  })
+  itemTypesAccepted: PropTypes.arrayOf(PropTypes.string).isRequired
 }
 
-export default memo(FilePickerBody)
+export default FilePickerBody
```

Comment:
nit: let's take advantage of this PR to create a FilePickerBody.spec with a sanity test

Reply:
I'll have to move on some other points, we can improve this in the next update 👍